crates/sui-core/src/authority.rs

```diff
@@ -2399,6 +2399,30 @@ impl AuthorityState {
             .compute_object_reference())
     }
 
+    pub fn get_authenticator_state_start_version(&self) -> SuiResult<Option<SequenceNumber>> {
```

Comment:
Is this function used anywhere?

Reply:
no, was from an earlier version of the PR, thanks for the catch